src/client.rs

```diff
@@ -1198,7 +1199,7 @@ impl From<u64> for PointId {
     }
 }
 
-#[derive(Clone, PartialEq)]
+#[derive(Clone, PartialEq, Debug, Default, Serialize, Deserialize)]
```

Comment:
The derive isn't producing that much code. Perhaps just (de)serializing via `serde_json::Map`'s ``iter()` and `collect()` could allow us to remove the `derive`, thus saving a few seconds build time?